lib/bn.js

```diff
@@ -53,6 +53,12 @@ else
 BN.BN = BN;
 BN.wordSize = 26;
 
+try {
+  BN._Buffer = require('buf' + 'fer').Buffer;
```

Comment:
@indutny @Kagami I've came across this version of instructing browser of non inclusion (from here: https://github.com/jprichardson/secure-random/commit/bccdbd2004c0895f8a3a382db19f5aac7643a6a1), might worth a try?

It works by adding a section to `package.json`